specs/Print.md

```diff
@@ -0,0 +1,473 @@
+  HRESULT Invoke([in] HRESULT errorCode, [in] IStream* pdfStream);
+}
+
+/// Settings used by the `PrintWithSettings` method.
+[uuid(DA3AD25C-D121-44DC-83DA-BF5E50DBF0DF), object, pointer_default(unique)]
+interface ICoreWebView2PrintSettings2 : ICoreWebView2PrintSettings {
+  /// Page range to print. Defaults to empty string, which means print all pages.
+  ///
+  /// The PageRanges property is a list of page ranges specifying one or more pages that should be printed separated by commas. Any whitespace between page ranges is ignored.
+  /// A valid page range is either a single integer identifying the page to print, or a range in the form `[start page]-[last page]` where `start page` and `last page` are integers identifying the first and last inclusive pages respectively to print. Every page identifier is an integer
+  /// greater than 0 unless wildcards are used (see below examples). The first page is 1.
+  ///
+  /// In a page range of the form `[start page]-[last page]` the start page number must be larger than 0 and less than or equal to the document's total page count.
+  /// If the `start page` is not present, then 1 is used as the `start page`.
+  /// The `last page` must be larger than the `start page`.
+  /// If the `last page` is not present, then the document total page count is used as the `last page`.
+  ///
+  /// If page range is not valid or if a page is greater than document total page count,
+  /// `ICoreWebView2PrintWithSettingsCompletedHandler` or ICoreWebView2PrintToPdfStreamCompletedHandler` handler will return `E_ABORT`.
+  ///
+  /// |       Example       | Valid |               Notes                   |
+  /// "2"                   |  Yes  | Assuming document total page count >= 2.
+  /// "1-4, 9, 3-6, 10, 11" |  Yes  | Assuming document total page count >= 11.
+  /// "1-4, -6"             |  Yes  | Assuming document total page count >= 6.
+  /// "2-"                  |  Yes  | Assuming document total page count >= 2, means from 2 to the * end.
+  /// "4-2, 11, -6"         |  No   |
+  ///  "-"                  |  Yes  | Assuming document total page count >= 1.
+  ///  "1-4dsf, 11"         |  No   | Regardless of document total page count.
+  ///
+  /// The caller must free the returned string with `CoTaskMemFree`. See
+  /// [API Conventions](/microsoft-edge/webview2/concepts/win32-api-conventions#strings)
+  [propget] HRESULT PageRanges([out, retval] LPWSTR* value);
+
+  /// Set the `PageRanges` property.
+  [propput] HRESULT PageRanges([in] LPCWSTR value);
+
+  /// Number of pages per sheet. Minimum value is `1` and the maximum is `16`.
+  /// The default value is 1
+  [propget] HRESULT PagesPerSheet([out, retval] INT32* value);
+
+  /// Set the `PagesPerSheet` property. Returns `E_INVALIDARG` if an invalid value is
+  /// provided, and the current value is not changed.
+  [propput] HRESULT PagesPerSheet([in] INT32 value);
+
+  /// Number of copies to print. Minimum value is `1` and the maximum copies count is `999`.
```

Comment:
why is there a max?

Reply:
This is to match browser print preview dialog max copies count.